docs/installation/requirements.md

```diff
@@ -122,7 +122,7 @@ Below are the supported limits for virtual machines on XCP-ng.
 
 - **Virtual CPUs (vCPUs) per VM**:
   - For untrusted VMs, the security-supported limit is **32 vCPUs**.
-  - For trusted VMs, the theoretical limit is **254 vCPUs**. Linux VMs with **128 vCPUs** have been successfully tested.
+  - For trusted VMs, the theoretical limit is **254 vCPUs**. Linux VMs with **128 vCPUs** in BIOS mode and **96 vCPUs** in UEFI mode have been successfully tested. Developments are planned to increase these limitations.
```

Comment:
Still, this doesn't really describe reality, IMO. 

Shouldn't it be: 


```suggestion
  - For trusted VMs, the theoretical limit is **254 vCPUs** in BIOS mode (Linux VMs with **128 vCPUs** have been successfully tested) and **96 vCPUs**. Developments are planned to bring UEFI VM limits to the level of BIOS VM limits.
```

Reply:
I don't think we have tested with 129 vCPUs in bios mode. Have we? That's why we say we tested 128. 254 could work, but requires the hardware to do the test.